src/app/in-the-news/mind-share-detail.component.html

```diff
@@ -9,17 +9,25 @@ <h1>{{mindShare.title}}</h1>
       <div class="details">
         <img class="rounded-circle"
              [src]="'./assets/authors/' + mindShare.authorImageUrl" />
-        <div class="author-info">
+        <div *ngIf="!mindShare.isPublication" class="author-info">
           <span>By <b>{{mindShare.authorName}}</b></span>
           <span class="divider"> / </span>
           <span>{{mindShare.publishDate | date: 'shortDate'}}</span>
         </div>
-        <div class="publish-info">
+        <div *ngIf="!mindShare.isPublication" class="publish-info">
           Originally Published By
           <a [href]="mindShare.publishUrl" target="_blank">
             {{mindShare.publishName}}
           </a>
         </div>
+        <div *ngIf="mindShare.isPublication" class="author-info">
```

Comment:
So I want to reduce some of the code duplication:

- For the author-info section, let's just remove the "By" text.  We won't need to differentiate between a publication then.
- For publish-info section, let's put the logic inside the class, so
```
    <div class="publish-info">
        ... code for spcificiation between isPublication and !isPublication
    </div>
```